src/configs/Reports.config.js

```diff
@@ -121,80 +133,70 @@ const Reports = (filter) => ({
         },
         {
           key: `reports_for_${filter}`,
-          selector: (reports) => reports.total_items,
-        },
-      ],
-    },
-    {
-      Card: ReportTile,
-      shouldRender: () => true,
-      name: "visitors",
-      text: () => ({
-        title: __("Visitors", "wp-module-ecommerce"),
-      }),
-      state: {
-        reportValue: getReportValue(`analytics_for_${filter}`),
-        change: calculateDelta(`analytics_for_${filter}`),
-      },
-      queries: [
-        {
-          key: "pluginStatus",
-          selector: (plugins) =>
-            PluginsSdk.queries.isPlugin(
-              plugins,
-              ["woocommerce", "jetpack"],
-              "active"
-            ),
-        },
-        {
-          key: `analytics_for_${filter}`,
-          selector: (stats) => stats.visitors,
-        },
-      ],
-    },
-    {
-      Card: ReportTile,
-      shouldRender: () => true,
-      name: "views",
-      text: () => ({
-        title: __("Views", "wp-module-ecommerce"),
-      }),
-      state: {
-        reportValue: getReportValue(`analytics_for_${filter}`),
-        change: calculateDelta(`analytics_for_${filter}`),
-      },
-      queries: [
-        {
-          key: "pluginStatus",
-          selector: (plugins) =>
-            PluginsSdk.queries.isPlugin(
-              plugins,
-              ["woocommerce", "jetpack"],
-              "active"
-            ),
-        },
-        {
-          key: `analytics_for_${filter}`,
-          selector: (stats) => stats.views,
+          selector: (reports) => reports.total_sales,
         },
       ],
     },
+    // {
+    //   Card: ReportTile,
+    //   shouldRender: () => true,
+    //   name: "net_sales",
+    //   text: () => ({
+    //     title: __("Net Sales", "wp-module-ecommerce"),
+    //   }),
+    //   state: formatMoneyToTile(filter),
+    //   queries: [
+    //     { key: "currency" },
+    //     {
+    //       key: "pluginStatus",
+    //       selector: (plugins) =>
+    //         PluginsSdk.queries.isPlugin(plugins, ["woocommerce"], "active"),
+    //     },
+    //     {
+    //       key: `reports_for_${filter}`,
+    //       selector: (reports) => reports.net_sales,
+    //     },
+    //   ],
+    // },
+
+    // {
+    //   Card: ReportTile,
+    //   shouldRender: () => true,
+    //   name: "products",
+    //   text: () => ({
+    //     title: __("Products Sold", "wp-module-ecommerce"),
+    //   }),
+    //   state: {
+    //     reportValue: getReportValue(`reports_for_${filter}`),
+    //     change: calculateDelta(`reports_for_${filter}`),
+    //   },
+    //   queries: [
+    //     {
+    //       key: "pluginStatus",
+    //       selector: (plugins) =>
+    //         PluginsSdk.queries.isPlugin(plugins, ["woocommerce"], "active"),
+    //     },
+    //     {
+    //       key: `reports_for_${filter}`,
+    //       selector: (reports) => reports.total_items,
+    //     },
+    //   ],
+    // },
+
```

Comment:
can we remove commented code, if it is not required ?

Reply:
@manikantakailasa ^

Reply:
done